Cargo.toml

```diff
@@ -25,6 +25,8 @@ hashbrown = { version = "0.13", default-features = false, optional = true }
 digest = "0.10"
 derivative = { version = "2", features = [ "use_core" ] }
 rayon = { version = "1", optional = true }
+criterion = { version = "0.5", default-features = false, optional = true }
+rand_chacha = { version = "0.3.0", default-features = false, optional = true }
```

Comment:
hm we don't want `criterion` as a normal dependency, as it has too many dependencies. `criterion` should be a dev-dependency.

Reply:
Also, `rand_chacha` is already a dev-dependency.